R/pkg/inst/tests/testthat/test_mllib.R

```diff
@@ -761,27 +761,13 @@ test_that("spark.kstest", {
   expect_equal(stats$p.value, rStats$p.value, tolerance = 1e-4)
   expect_equal(stats$statistic, unname(rStats$statistic), tolerance = 1e-4)
 
-  printStr <- print.summary.KSTest(testResult)
```

Comment:
The output is relevant to the implementation of ```KolmogorovSmirnovTestResult.toString```, if it was updated, this test will be broken which is not reasonable IMO. We have already checked the summary output value in unit test, I think it's not necessary to check another kinds of these value again. And I believe we will add more summary information later, which will require updating the expected printing summary again and again. In all other ML wrappers, we did not check print summary.